src/ui/src/app/modules/home/pages/landing/landing.component.html

```diff
@@ -1,9 +1,4 @@
-<app-nav-bar></app-nav-bar>
-<div class="parallax">
-  <app-amongus-parallax></app-amongus-parallax>
-  <div class="headertextcontainer">
-    <h1 class="amogush">AMOGUS</h1>
-  </div>
+<div class="landing-container">
+    <app-amogus-button text="Play" link="/user/game"></app-amogus-button>
```

Comment:
falsche route http://localhost:4200/user/game-selection

Reply:
Jasmin meinte sie hat btw auch so Icons für die buttons rausgesucht. Die müssten auch noch dazu kommen

Reply:
Is im discord angepinnt